Dockerfile

```diff
@@ -40,7 +40,7 @@ WORKDIR /home/epoch/node
 # Erl handle SIGQUIT instead of the default SIGINT
 STOPSIGNAL SIGQUIT
 
-EXPOSE 3013 3015 3113 3114
+EXPOSE 3013 3014 3015 3113 3114
```

Comment:
Could you please integrate [doc on how to run localnet with docker compose and state channels](https://github.com/aeternity/epoch/blob/061e7976aa061b30ddc9afdfc85defe591125b87/docs/docker.md#localnet)?

@dincho Is any websocket endpoint already supported in docker compose and [documented](https://github.com/aeternity/epoch/blob/061e7976aa061b30ddc9afdfc85defe591125b87/docs/docker.md#localnet)?

Reply:
@lucafavatella Yes, we support WS API on port 3114 for localnet - [configured in nginx proxy](https://github.com/aeternity/epoch/blob/master/docker/nginx-default.conf#L14-L23), it's not documented tho.

Support for this WS can be added in similar fashion with copy pasting, although a common WS config duplicated should be factored out.